src/post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+import { Entry } from "contentful";
+import * as React from "react";
+
+import PostDataSource from "datasources/PostsDataSource";
+import { IImage } from "model/common/Iimage";
+
+import "post/post.css";
+
+export interface IPost {
+    title: string;
+    description: string;
+    image: Entry<IImage>;
+    author: string;
+    paragraph: string;
+    paragraph2: string;
+    paragraph3: string;
+    paragraph4: string;
+    publishedDate: string;
+}
+
+interface IState {
+    postContents?: IPost;
+}
+
+class Post extends React.Component<{}, IState>{
+    public state: IState = {};
+
+    public render(): JSX.Element {
+        const { postContents } = this.state;
+        if (!postContents) {
+            return null;
+        }
+
+        return (
+            <div className="post-container">
+                <h1>{postContents.title}</h1>
+                <h4>{postContents.description}</h4>
+                <img
+                    src={postContents.image.fields.file.url}
+                    alt={postContents.image.fields.description}
+                />
+                <div>
+                    <p>{postContents.paragraph}</p>
```

Comment:
It should be dynamic. You should fix this. The content type should have the field that shows text-editor where you can write lots of text and can format it too.  Here you can write only 4 paragraphs. What if we have more paragraphs?

Reply:
okay, will work on this, Thanks.